### ISSUE TITLE
Sometimes, "Add a new Stack" button doesn't work

### ISSUE THREAD

**Issue by LollenJumplan** [nextcloud/deck#89]
### Steps to reproduce

1. Create new Deck
2. Type name for a new Stack
3. Click the "+" (Enter seems to always work)

### Expected behaviour
New Stack is always created

### Actual behaviour
Often, the "+" disappears and nothing else happens. Reloading the page, you can see that the stack has not been created (so it seems it is not only a display issue).

### Server configuration
**Operating system**:
debian testing

**Web server:**
apache 2.4.25-3

**Database:**
mysql 5.6.30-1

**PHP version:**
7.1.1-1

**Server version:** (see your admin page)
nextcloud 11.0.2

**Deck version:** (see the apps page)
0.1.2

**Updated from an older installed version or fresh install:**
yes

```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
No errors have been found.
```

**List of activated apps:**
Enabled:
  - activity: 2.4.1
  - admin_audit: 1.1.0
  - announcementcenter: 3.0.0
  - apporder: 0.3.3
  - calendar: 1.5.2
  - comments: 1.1.0
  - contacts: 1.5.3
  - dav: 1.1.1
  - deck: 0.1.2
  - direct_menu: 0.10.1
  - federatedfilesharing: 1.1.1
  - federation: 1.1.1
  - files: 1.6.1
  - files_accesscontrol: 1.1.2
  - files_pdfviewer: 1.0.1
  - files_sharing: 1.1.1
  - files_texteditor: 2.2
  - files_trashbin: 1.1.0
  - files_versions: 1.4.0
  - files_videoplayer: 1.0.0
  - firstrunwizard: 2.0
  - gallery: 16.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - nextcloud_announcements: 1.0
  - notes: 2.2.0
  - notifications: 1.0.1
  - password_policy: 1.1.0
  - provisioning_api: 1.1.0
  - serverinfo: 1.1.1
  - sharebymail: 1.0.1
  - spreed: 1.2.0
  - survey_client: 0.1.5
  - systemtags: 1.1.3
  - tasks: 0.9.5
  - templateeditor: 0.2
  - theming: 1.1.1
  - twofactor_backupcodes: 1.0.0
  - updatenotification: 1.1.1
  - workflowengine: 1.1.1

**The content of config/config.php:**

```
{
    "system": {
        "memcache.local": "\\OC\\Memcache\\APCu",
        "updatechecker": true,
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "11.0.2.7",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "ssl",
        "mail_from_address": "owncloud",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "loglevel": 0,
        "theme": "",
        "updater.release.channel": "stable",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

**Comment by dirigit** [comment]
My testinstallation shows same symptom and:

I cannot see new stacks at all.

To be more precise:
I don't see any stack, "Create new stack" only.

**Comment by x9t9** [comment]
I can also confirm this issue .
Pressing `enter `works . Clicking with mouse on `"+"` sign does not . 
I do not have my server configuration here at hand , but if necessary I will post .

Tested on : Vivaldi , Chrome , Firefox - latest .

**Comment by TGX2010** [comment]
Unsubscribe me from this thread!
Thanks!
Toni Neumann 

-------- Original message --------

**Comment by x9t9** [comment]
@TGX2010 - If you get notified on all conversations unwillingly - I believe that you get this because of your settings - you are probably  "**watching**" this repository . Change your preferences to avoid recieving further notifications .

**Comment by juliushaertl** [comment]
Fixed with #95
